keyboards/cannonkeys/aella/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Aella",
+    "url": "https://www.alchemistkeyboards.com/aella",
+    "maintainer": "Alchemist Keyboards",
+    "width": 16,
+    "height": 6,
+    "layouts": {
+        "LAYOUT": {
```

Havent checked the actual content, but the names dont key together correctly
```suggestion
        "LAYOUT_all": {
```